docs/generate_extension_cat_db.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
```

Comment:
Can you add a top-level comment explaining what this does?

Reply:
comment added in new location of code